clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -652,19 +652,26 @@ def DynamicCastInfoAttr
 // AddressSpaceAttr
 //===----------------------------------------------------------------------===//
 
-// TODO: other CIR AS cases
-def AS_Target : I32EnumAttrCase<"target", 21>;
+def AS_OffloadPrivate : I32EnumAttrCase<"offload_private", 1>;
+def AS_OffloadLocal : I32EnumAttrCase<"offload_local", 2>;
+def AS_OffloadGlobal : I32EnumAttrCase<"offload_global", 3>;
+def AS_OffloadConstant : I32EnumAttrCase<"offload_constant", 4>;
+def AS_OffloadGeneric : I32EnumAttrCase<"offload_generic", 5>;
+def AS_Target : I32EnumAttrCase<"target", 6>;
 
 def AddressSpaceAttr : CIR_Attr<"AddressSpace", "addrspace"> {
 
   let summary = "Address space attribute for pointer types";
   let description = [{
-    The address space attribute models `clang::LangAS` rather than the LLVM
-    address space, which means it's not yet converted by the address space map
-    to carry target-specific semantics.
+    The address space attribute is used in pointer types. It essentially models
+    `clang::LangAS` rather than the LLVM address space.
```

Comment:
Perhaps change `It essentially models clang::LangAS rather than the LLVM address space` to `It essentially provides an unified models on top of clang::LangAS, rather than LLVM address spaces`